shaders_js.go

```diff
@@ -0,0 +1,83 @@
+//+build netgo
+
+package engo
+
+import (
+	"github.com/gopherjs/gopherjs/js"
+)
+
+type Shader interface {
+	Initialize(width, height float32)
+	Pre()
+	Draw(texture *js.Object, buffer *js.Object, x, y, rotation float32)
```

Comment:
It's `js.Object` under the hood, but the webgl *defines* `Texture`. 

https://github.com/EngoEngine/gl/blob/master/gl_webgl.go#L16

This is `EngoEngine.gl.Texture`. Which is what I am referring to.